src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -649,6 +651,31 @@ public void SetLastCompletedClusterTransactionIndex(DocumentsOperationContext co
                 tree.Add(LastCompletedClusterTransactionIndexSlice, indexSlice);
         }
 
+        public static long ReadLastRevisionsBinCleanerState(Transaction tx)
```

Comment:
we do have RevisionsStorage, wouldn't it be a better place?

Reply:
`GlobalTreeSlice` and all other global slices are private, do you want to make them public and move the method to `RevisionsStorage`?

Reply:
Done